.pre-commit-config.yaml

```diff
@@ -34,9 +34,9 @@ repos:
     hooks:
     - id: update_copyright
       name: update_copyright
-      entry: python scripts/copyright_update.py
+      entry: python3 scripts/copyright_update.py
       language: system
-      types: [python]
+      types: [python3]
 
```

Comment:
Hmm, @je-cook might comment, but this looks more like an environment alias problem. This is likely to break stuff for other people.

Reply:
If your environment is active python will always link to python3. Also python2 is severely out of date now and most linux distros are now linking python to python3 by default. I think we should follow that trend so leave it as `python`